tasks/oracle_job.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: "Cohesity Protection Job: Set {{ cohesity_oracle.job_name | default('job_name') }} to state of {{ cohesity_oracle.state | default('present') }}"
```

Comment:
As anvesh pointed out , make sure you create docs for the playbooks.

Reply:
Okay Ashish.